common-react/@dbeaver/react-tests/package.json

```diff
@@ -30,7 +30,7 @@
     "@testing-library/dom": "^10",
     "@testing-library/jest-dom": "^6",
     "@testing-library/react": "^16",
-    "@types/node": "^22",
+    "@types/node": "^24",
```

Comment:
Please keep the `^22` version for node; it's our current minimal supported node version